requirements/env_common.yml

```diff
@@ -26,3 +26,4 @@ dependencies:
     - xgboost
     - git+https://github.com/samoturk/mol2vec
     - tb-nightly # @arunppsg shift to tensorboard stable version once tb 2.9.1 is released
+    - pyscf
```

Comment:
What is this needed for?  Adding a new dependency to DeepChem should be done only when absolutely necessary and needs to be carefully considered first.

Reply:
Yeah I wanted to discuss this over in the developer calls as well. I thought this was necessary as ferminet needs a baseline solution of Hartree-Fock for pre-training which can be obtained from pySCF.

Reply:
We can add it as a soft dependency, no? 

I think it may be better to use the pyscf conda channel to install over installing through pip.